enhancements/cloud-integration/infrastructure-external-platform-type.md

```diff
@@ -0,0 +1,338 @@
+---
+title: infrastructure-external-platform-type
+authors:
+- "@lobziik"
+- "@elmiko"
+reviewers:
+- "@dhellmann"
+- "@mhrivnak"
+- "@deads2k" - API change, KCM, library-go
+- "@JoelSpeed" - cluster infra
+- "@jsafrane" - cluster storage operator
+- "@dmage" - image registry
+approvers:
+- TBD
+api-approvers:
+- "@deads2k"
+- "@JoelSpeed"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link:
+- [OCPPLAN-9429](https://issues.redhat.com/browse/OCPPLAN-9429)
+- [OCPPLAN-8156](https://issues.redhat.com/browse/OCPPLAN-8156)
+see-also:
+- [OCP infrastructure provider onboarding guide](https://docs.providers.openshift.org/overview/)
+- [Out-of-tree cloud provider integration support](https://github.com/openshift/enhancements/blob/master/enhancements/cloud-integration/out-of-tree-provider-support.md)
+- [Platform Operators Proposal](https://github.com/openshift/enhancements/pull/1170)
+- [Capabilites selection](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md)
+---
+
+# Introduce new platform type "External" in the OpenShift specific Infrastructure resource
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This EP describes an addition of the new special platform type, referred here as "External".
+
+Such new platform type would serve as a signal when OpenShift is running on an infrastructure platform
```

Comment:
this reads a little odd, i might start this paragraph like "The new External platform type would ...."

i think we should make it clear that the supplemental components will be infrastructure specific things that are designed for the platform. i want to make sure this doesn't get confused with more general platform operators, although a provider is free to include those as well.